.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDatePeriodExtractor.cs

```diff
@@ -527,6 +527,27 @@ private List<Token> MatchComplexCases(string text, List<ExtractResult> simpleDat
         {
             var er = this.config.DatePointExtractor.Extract(text, reference);
 
+            if (er.Count == 0)
+            {
+                // Handle "now"
```

Comment:
You can change this comment to be something like 
// Handle "now" as a possible date point for ranges, as it's a datetime.
A little more descriptive of the why.

Reply:
get it!